test/unit/lib/plugins/aws/package/compile/events/alb/lib/validate.test.js

```diff
@@ -258,4 +258,20 @@ describe('#validate()', () => {
       );
     });
   });
+
+  describe('#validateTargetGroupName()', () => {
```

Comment:
Let's remove this tests (all tests should be configured via `runServerless` util -> https://github.com/serverless/serverless/tree/master/test#unit-tests

Also I'd keep all tests in `index.test.js` (this file is leftover of legacy tests, which we didn't refactor yet).

Summarizing, we can add another test in `index.test.js` but which has its own `runServerless` run configured. Then we should test the result error code. as e.g. it's done here: https://github.com/serverless/serverless/blob/0e6a1ce2d4a28a480683192cc6ac639c7bfe0d8b/test/unit/lib/plugins/aws/package/compile/events/httpApi.test.js#L541-L573

Reply:
I tested both cases, with a `targetGroupPrefix` to check that it's rejected and the nominal case with the Name property being correctly set